content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -19,6 +19,59 @@ further_reading:
 
 Find below the different initialization options available with the [Datadog Browser SDK][1].
 
+### Identify user sessions
+Adding user information to your RUM sessions makes it easy to:
+* Follow the journey of a given user
+* Know which users are the most impacted by errors
+* Monitor performance for your most important users
+
+{{< img src="real_user_monitoring/browser/advanced_configuration/user-api.png" alt="User API in RUM UI"  >}}
+
+The following attributes are **optional** but it is recommended to provide **at least one** of them:
+
+| Attribute  | Description                                                                                              |
+|------------|----------------------------------------------------------------------------------------------------------|
+| usr.id    | Unique user identifier.                                                                                  |
+| usr.name  | User friendly name, displayed by default in the RUM UI.                                                  |
+| usr.email | User email, displayed in the RUM UI if the user name is not present. It is also used to fetch Gravatars. |
+
+To identify user sessions, use the `setUser` API:
+
+{{< tabs >}}
+{{% tab "NPM" %}}
+```javascript
+datadogRum.setUser({
+    id: 1234,
+    name: John Doe,
+    email: john@doe.com
+})
+```
+
+{{% /tab %}}
+{{% tab "CDN async" %}}
+```javascript
+DD_RUM.onReady(function() {
+    DD_RUM.setUser({
+        id: 1234,
```

Comment:
Since the facet is typed as string, we have typed the id as string.
If a number is provided, the SDK will convert it to string.
It could be better to use a string in the example then, wdyt?

Reply:
Indeed, changed it